### ISSUE TITLE
Bug: Not able to parse codeforces 1702  C (wrong problem input)

### ISSUE THREAD

**Issue by michelececcacci** [jmerle/competitive-companion#288]
Platform: Chromium `103.0.5060.53 (Official Build) (64-bit) `
Using competitive companion 2.36.0 on https://codeforces.com/contest/1702/problem/C to parse the problem's input produces the following output: 
```html
3
<div class="test-example-line test-example-line-odd test-example-line-1" style="">6 3</div><div class="test-example-line test-example-line-odd test-example-line-1" style="">3 7 1 5 1 4</div><div class="test-example-line test-example-line-odd test-example-line-1" style="">3 5</div><div class="test-example-line test-example-line-odd test-example-line-1" style="">1 7</div><div class="test-example-line test-example-line-odd test-example-line-1" style="">3 10</div><div class="test-example-line test-example-line-even test-example-line-2" style=""><div class="test-example-line test-example-line-even test-example-line-2" style="">3 3</div><div class="test-example-line test-example-line-even test-example-line-2" style="">1 2 1</div><div class="test-example-line test-example-line-even test-example-line-2" style="">2 1</div><div class="test-example-line test-example-line-even test-example-line-2" style="">1 2</div><div class="test-example-line test-example-line-even test-example-line-2" style="">4 5</div><div class="test-example-line test-example-line-odd test-example-line-3" style=""><div class="test-example-line test-example-line-odd test-example-line-3" style="">7 5</div><div class="test-example-line test-example-line-odd test-example-line-3" style="">2 1 1 1 2 4 4</div><div class="test-example-line test-example-line-odd test-example-line-3" style="">1 3</div><div class="test-example-line test-example-line-odd test-example-line-3" style="">1 4</div><div class="test-example-line test-example-line-odd test-example-line-3" style="">2 1</div><div class="test-example-line test-example-line-odd test-example-line-3" style="">4 1</div><div class="test-example-line test-example-line-odd test-example-line-3" style="">1 2</div><div class="test-example-line test-example-line-odd test-example-line-3" style=""></div></div></div>
6 3
3 7 1 5 1 4
3 5
1 7
3 10
<div class="test-example-line test-example-line-even test-example-line-2" style="">3 3</div><div class="test-example-line test-example-line-even test-example-line-2" style="">1 2 1</div><div class="test-example-line test-example-line-even test-example-line-2" style="">2 1</div><div class="test-example-line test-example-line-even test-example-line-2" style="">1 2</div><div class="test-example-line test-example-line-even test-example-line-2" style="">4 5</div><div class="test-example-line test-example-line-odd test-example-line-3" style=""><div class="test-example-line test-example-line-odd test-example-line-3" style="">7 5</div><div class="test-example-line test-example-line-odd test-example-line-3" style="">2 1 1 1 2 4 4</div><div class="test-example-line test-example-line-odd test-example-line-3" style="">1 3</div><div class="test-example-line test-example-line-odd test-example-line-3" style="">1 4</div><div class="test-example-line test-example-line-odd test-example-line-3" style="">2 1</div><div class="test-example-line test-example-line-odd test-example-line-3" style="">4 1</div><div class="test-example-line test-example-line-odd test-example-line-3" style="">1 2</div><div class="test-example-line test-example-line-odd test-example-line-3" style=""></div></div>
3 3
1 2 1
2 1
1 2
4 5
<div class="test-example-line test-example-line-odd test-example-line-3" style="">7 5</div><div class="test-example-line test-example-line-odd test-example-line-3" style="">2 1 1 1 2 4 4</div><div class="test-example-line test-example-line-odd test-example-line-3" style="">1 3</div><div class="test-example-line test-example-line-odd test-example-line-3" style="">1 4</div><div class="test-example-line test-example-line-odd test-example-line-3" style="">2 1</div><div class="test-example-line test-example-line-odd test-example-line-3" style="">4 1</div><div class="test-example-line test-example-line-odd test-example-line-3" style="">1 2</div><div class="test-example-line test-example-line-odd test-example-line-3" style=""></div>
7 5
2 1 1 1 2 4 4
1 3
1 4
2 1
4 1
1 2
```
Expected ouput would be something like:
```
3
6 3
3 7 1 5 1 4
3 5
1 7
3 10
3 3
1 2 1
2 1
1 2
4 5
7 5
2 1 1 1 2 4 4
1 3
1 4
2 1
4 1
1 2
```
I could take a look at the issue myself, but i'm not really familiar with typescript

**Comment by jmerle** [comment]
Fixed in a36358531f418630ca6fb99b84300bffc0ce48d3, will be released within an hour. Thanks for reporting the issue!

**Comment by michelececcacci** [comment]
That was impressively fast, thanks a lot!
